jekyll/_cci2/migrating-from-aws.adoc

```diff
@@ -0,0 +1,246 @@
+= Migrating from AWS
+:page-layout: classic-docs
+:page-liquid:
+:icons: font
+:toc: macro
+:toc-title:
+:sectanchors:
+
+This document provides an overview of how to migrate from AWS CodeCommit to CircleCI. 
+
+NOTE: **Tips provided by ImagineX Consulting**
+
+== Source Control Setup
+If you are using AWS Code Commit, you will first need to migrate your source code to GitHub or BitBucket. See the following for details on how to import your code:
+
+* https://help.github.com/en/articles/importing-a-repository-with-github-importer[GitHub]
+* https://help.github.com/en/articles/importing-a-repository-with-github-importer[BitBucket]
+
+=== GitHub Enterprise
+
+Following are the steps required for using the git command line tool to import your code into GitHub Enterprise:
+
+. Create an empty repository on your GitHub Enterprise instance.
+. Create a bare clone of your external repository on your local machine, fetching all remote tags (refs/tags/\*) and copying all remote branch heads (refs/heads/\*) directly to their corresponding local branch heads.
++
+----
+git clone https://external-host.com/extuser/repo.git --bare
+----
+. Add your GitHub Enterprise repository as a remote reference in your local clone.
++
+----
+cd [repo-name]
+git remote add enterprise git@[hostname]:[owner]/[repo-name].git
+----
+. Push all local references (refs/*) up to your remote GitHub Enterprise repository.
++
+----
+git push enterprise --mirror
+----
+
+Once you have imported your code into GitHub or BitBucket, you can start creating a project in CircleCI using the https://circleci.com/docs/2.0/getting-started/[Getting Started guide].
+
+
+== Build Configuration
+
+Next, you will need to migrate your build configuration. On AWS CodeBuild, the build configuration is either defined in the web interface or in a file called `buildspec.yml` in the root directory of your source code repository. If you use shell scripts to perform your build, you can reuse those scripts in CircleCI.
+
+First, create a CircleCI build configuration file. In the root directory of your source code repository, create a folder named .circleci and create a file in that folder named config.yml. Next, follow the CircleCI documentation here to learn how to configure the .config.yml file.
+
+The AWS CodeBuild and CircleCI configurations will be different. It may be helpful to have both Azure DevOps and CircleCI reference documentation open side-by-side to help with the conversion of the build steps:
```

Comment:
```suggestion
The AWS CodeBuild and CircleCI configurations will be different. It may be helpful to have both AWS DevOps and CircleCI reference documentation open side-by-side to help with the conversion of the build steps:
```
Should this be `AWS` and not `Azure`?